en/views.rst

```diff
@@ -66,15 +66,58 @@ View Templates
 ==============
 
 The view layer of CakePHP is how you speak to your users. Most of the time your
-views will be showing (X)HTML documents to browsers, but you might also need to
+views will be rendering HTML/XHTML documents to browsers, but you might also need to
 reply to a remote application via JSON, or output a CSV file for a user.
 
-By default CakePHP template files are written in plain PHP and have a default
-extension of **.ctp** (CakePHP Template). These files contain all the
-presentational logic needed to get the data it received from the
-controller in a format that is ready for the audience you're
-serving to. If you'd prefer using a templating language like
-`Twig <http://twig.sensiolabs.org>`__, a subclass of View will bridge your templating
+CakePHP template files have a default extension of **.ctp** (CakePHP Template)
```

Comment:
@jeffblack360 Sorry to comment in a closed issue, but I noticed an undesirable change.

 In the above changes, a phrase `are written in plain PHP` was removed from the original text. But I think this is important for explaining what CakePHP Template is. Without this phrase, users cannot be sure whether CakePHP Template is just a plain PHP or yet another PHP-like template.

But this phrase seems to have removed intentionally. Why did you remove this phrase?